avocado_vt/loader.py

```diff
@@ -16,6 +16,7 @@
 Avocado VT plugin
 """
 
```

Comment:
I don't see the "refactor" mentioning in the commit message. It can't be easily split as the `vt-list` does not support some of the other options and sharing ~2 options as refactor and then extending it would look hideous (and the function names would be misleading in the first patch).